test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTest/.gitignore

```
@@ -1,3 +1,7 @@
 # Self-signed cert generated by integration test
 otel-collector.crt
 otel-collector.key
+otel-client.crt
+otel-client.key
+otel-untrusted-collector.crt
+otel-untrusted-collector.key
```

minor (this also applies to other files in the PR):

```suggestion
otel-untrusted-collector.key

```

---

Thx, fixed.